spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/reactive/DefaultPartHttpMessageReaderCustomizerConfiguration.java

```diff
@@ -0,0 +1,65 @@
+package org.springframework.boot.autoconfigure.web.reactive;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Paths;
+import java.util.function.Consumer;
+
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.context.properties.PropertyMapper;
+import org.springframework.boot.web.codec.CodecCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.annotation.Order;
+import org.springframework.http.codec.multipart.DefaultPartHttpMessageReader;
+import org.springframework.util.unit.DataSize;
+
+@Configuration(proxyBeanMethods = false)
+@EnableConfigurationProperties(MultipartProperties.class)
+class DefaultPartHttpMessageReaderCustomizerConfiguration {
+
+	@Bean
+	@Order(10)
+	CodecCustomizer defaultPartHttpMessageReaderCustomizer(MultipartProperties multipartProperties) {
+		return (configurer) -> configurer.defaultCodecs().configureDefaultCodec((codec) -> {
```

Comment:
Not super happy about our options to "get at" the `DefaultPartHttpMessageReader`. 

I chose to use an auto-configured customizer rather than insert a specific `Consumer<Object>` in [WebFluxAutoConfiguration.java](https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/reactive/WebFluxAutoConfiguration.java#L182) such as :
```java
		@Override
		public void configureHttpMessageCodecs(ServerCodecConfigurer configurer) {
             configurer.defaultCodecs().configureDefaultCodec((codec) -> { ... });
			this.codecCustomizers.orderedStream().forEach((customizer) -> customizer.customize(configurer));
		}
```
Seemed less intrusive and more auto-configed-like.

Reply:
Looks good to me!